### PR TITLE
issue template: clarify security/crash reporting

### PR DESCRIPTION
**Pull request by mattklein123**

### CODE REVIEW

**Comment by mattklein123** [comment]
cc @envoyproxy/security-team

File: ISSUES.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+**If you are reporting *any* crash or *any* potential security issue, *do not*
```
Comment (htuch) [inline ISSUES.md:1]: Why do we have both ISSUES_TEMPLATE.md and ISSUES.md after this PR @mattklein123?

Comment (htuch) [reply ISSUES.md:1]: I see, this is essentially a rough version of what GH now supports through explicit template choosers.

Comment (mattklein123) [reply ISSUES.md:1]: Right, this is pre new stuff. We can definitely make this all better.

**Pull request merged**